medusa/tv.py

```diff
@@ -1478,6 +1479,63 @@ def __unicode__(self):
         to_return += u'anime: {0}\n'.format(self.is_anime)
         return to_return
 
+    def to_json(self):
+        """Return JSON representation."""
+        # @TODO: Should we also return the season list with all episodes?
+        genres = list(set([v for v in self.genre.split('|') if v] + [v for v in self.imdb_info['genres'].replace('Sci-Fi', 'Science-Fiction').split('|') if v]))
+        indexers = {
```

Comment:
It'd be great if we had something like `indexer.find_by_id(1)` or `indexer.find_by_name('thetvdb')`and it'd return the indexer.